runelite-client/src/main/java/net/runelite/client/plugins/motherlode/MotherlodeSackOverlay.java

```diff
@@ -73,8 +69,7 @@ public Dimension render(Graphics2D graphics)
 
 		Widget sack = client.getWidget(WidgetInfo.MOTHERLODE_MINE);
 
-		panelComponent.getChildren().clear();
-		panelComponent.setBackgroundColor(ComponentConstants.STANDARD_BACKGROUND_COLOR);
+		panelComponent.setBackgroundColor(getPreferredColor());
```

Comment:
preferredColor can be null, just setting it to ComponentConstants.STANDARD_BACKGROUND_COLOR will achieve the same thing, see here: https://github.com/runelite/runelite/blob/master/runelite-client/src/main/java/net/runelite/client/ui/overlay/OverlayPanel.java#L94